cmd/providers.go

```diff
@@ -95,12 +99,19 @@ var cmdProvidersCreateOneLogin = &cobra.Command{
 			log.Fatalf(color.RedString("Provider '%s' already exists"), name)
 		}
 
+		switch region {
+		case "US", "EU":
+		default:
+			log.Fatal(color.RedString("Region must be one of US or EU"))
```

Comment:
Thanks. I would rephrase to `Region must be one of the following: US, EU` or `Region must be either US or EU`. Sounds better English-wise... Other than that - looks good to me.

Reply:
Seems smart, see latest